theano/tensor/tests/test_opt.py

```diff
@@ -3855,15 +3855,15 @@ def test_local_useless_tile(self):
         # But it isn't supported for now, so assert that we raise an
         # error.
 
-        self.assertRaises(ValueError, T.tile, v, (1,)*(v.ndim+1))
+        #self.assertRaises(ValueError, T.tile, v, (1,)*(v.ndim+1))
         # If the repeat parameter is shorter then m.ndim, it should
         # pad tot he left the repeat patter with 1. It is not supported for now.
```

Comment:
Right, `long` does not exist in Python 3 (all ints are long ints). You con use `six.integer_types` instead.